vlib/v/checker/tests/incorrect_name_sum_type.out

```diff
@@ -1,12 +1,18 @@
 vlib/v/checker/tests/incorrect_name_sum_type.vv:1:1: error: sum type `integer` must begin with capital letter
-    1 | type integer = i8 | i16 | int | i64
+    1 | type integer = i16 | i64 | i8 | int
       | ~~~~~~~~~~~~
-    2 | type Integer = i8 | i16 | int | i64
+    2 | type Integer = i16 | i64 | i8 | int
     3 |
-vlib/v/checker/tests/incorrect_name_sum_type.vv:4:1: error: method overrides built-in sum type method
-    2 | type Integer = i8 | i16 | int | i64
-    3 | 
-    4 | fn (i Integer) type_name() {
-      | ~~~~~~~~~~~~~~~~~~~~~~~~~~
+vlib/v/checker/tests/incorrect_name_sum_type.v:4:1: error: method overrides built-in sum type method
+    2 | type Integer = i16 | i64 | i8 | int
+    3 |
+    4 | fn (i Integer) type_idx() {
+      | ~~~~~~~~~~~~~~~~~~~~~~~~~
     5 | }
     6 |
+vlib/v/checker/tests/incorrect_name_sum_type.v:7:1: error: method overrides built-in sum type method
```

Comment:
```suggestion
vlib/v/checker/tests/incorrect_name_sum_type.vv:7:1: error: method overrides built-in sum type method
```

Reply:
Or just run `VAUTOFIX=1 ./v vlib/v/compiler_errors_test.v` 2 times, and commit the result.